atc/validate_test.go

```diff
@@ -1005,15 +1016,15 @@ var _ = Describe("ValidateConfig", func() {
 
 				It("throws a validation error", func() {
 					Expect(errorMessages).To(HaveLen(1))
-					Expect(errorMessages[0]).To(ContainSubstring("jobs.some-other-job.plan[0].get.some-resource.ensure.put.custom-name refers to a resource that does not exist ('some-missing-resource')"))
+					Expect(errorMessages[0]).To(ContainSubstring("jobs.some-other-job.plan[0].get.some-resource.error.put.custom-name refers to a resource that does not exist ('some-missing-resource')"))
 				})
 			})
 
-			Context("when a plan has an invalid step within an abort", func() {
```

Comment:
Looks like the test for `abort` went away? Or is this just a confusing diff?

Reply:
Confusing diff. Abort test is here: https://github.com/amanw/concourse/blob/master/atc/validate_test.go#L974

Reply:
Link to right branch: https://github.com/amanw/concourse/blob/on_error/atc/validate_test.go